packages/node/src/integrations/http.ts

```diff
@@ -103,7 +103,7 @@ export const instrumentHttp = Object.assign(
       },
 
       ignoreIncomingRequestHook: request => {
-        const url = getRequestUrl(request);
+        const url = request.url;
```

Comment:
just checking to be sure, is the url here the full URL, including protocol? so e.g. `http://...`? As we want these to be full URLs I guess always, if possible.

Reply:
xref https://github.com/getsentry/sentry-javascript/pull/12929#discussion_r1679081518

Reply:
@mydea `url` is NOT the full url, it is instead just the path (and maybe the query params? EDIT: definitely has query params too.).

the `node:IncomingMessage` type doesn't have a `protocol`, `hostname`, or `port` attr.
* For protocol you'd have to do something like access `req.socket.encrypted` and infer true === https
* For hostname you'd have to parse the request headers for a Host header, falling back to req.socket.address() if it doesn't exist.
* For port you'd have to access `req.socket.address().port`, I think.

So all that to say, sadly it's difficult to reconstruct the full url. Thoughts?

Screenshot:
<img width="869" alt="Screen Shot 2024-07-16 at 5 42 36 AM" src="https://github.com/user-attachments/assets/8669ca54-aee6-4f19-b02c-e9d5096f67ae">

Reply:
Here's all the available attrs:
<img width="512" alt="Screen Shot 2024-07-16 at 5 40 06 AM" src="https://github.com/user-attachments/assets/56eb8948-445d-4ca6-a61e-4146a730b76a">

Reply:
@mydea sorry i saw you moved your comment, so I just moved mine before I saw you replied 😅

Reply:
> Yeah, I feared that. IMHO this is OK, but we should maybe reflect this in the jsdoc/variables used for ignoreIncomingRequests, e.g. make it ignoreIncomingRequests(urlPath: string) or something like this?

Sounds reasonable to me but sadly this is a breaking change, no?

Reply:
Just changing the name of the variable is fine. Will make the change and update JSDoc and docs to reflect it. "Luckily" this never worked in v8 in the first place (afaict) so we're likely not behaviourally breaking folks.

Reply:
Regarding docs update: I'll do this in combination with the changes from #12930

Reply:
sounds good to me 🚀

Reply:
Done: [9348717](https://github.com/getsentry/sentry-javascript/pull/12929/commits/93487171df86f0befc955882b2519759e3306532)